addons/im_livechat/models/im_livechat_channel.py

```diff
@@ -95,6 +97,15 @@ def _compute_available_operator_ids(self):
         for livechat_channel in self:
             livechat_channel.available_operator_ids = operators_by_livechat_channel[livechat_channel]
 
+    @api.constrains("review_link")
+    def _check_review_link(self):
+        for record in self.filtered("review_link"):
+            url = urlparse(record.review_link)
+            if not (url.scheme in ("http", "https") and url.netloc):
+                raise ValidationError(
+                    self.env._("Invalid URL '%s'. The Review Link must start with 'http://' or 'https://'.") % record.review_link
+                )
```

Comment:
Can we write a simple test for this scenario?
- create live chat channel
- set link to javascript:alert('...') raises
- set link to incomplete URL with correct scheme (missing netloc) raises
- set link to valid URL works